itests/org.openhab.binding.tradfri.tests/itest.bndrun

```diff
@@ -0,0 +1,62 @@
+-include: ../itest-common.bndrun
+
+Bundle-SymbolicName: ${project.artifactId}
+Fragment-Host: org.openhab.binding.tradfri
+
+-runrequires: bnd.identity;id='org.openhab.binding.tradfri.tests'
+
+#
+# done
+#
+-runbundles: \
+	javax.measure.unit-api;version='[1.0.0,1.0.1)',\
+	org.apache.commons.io;version='[2.2.0,2.2.1)',\
+	org.apache.commons.lang;version='[2.6.0,2.6.1)',\
+	org.apache.felix.configadmin;version='[1.9.8,1.9.9)',\
+	org.apache.felix.scr;version='[2.1.10,2.1.11)',\
+	org.osgi.service.event;version='[1.4.0,1.4.1)',\
+	tec.uom.lib.uom-lib-common;version='[1.0.2,1.0.3)',\
+	tec.uom.se;version='[1.0.8,1.0.9)',\
+	osgi.enroute.hamcrest.wrapper;version='[1.3.0,1.3.1)',\
+	osgi.enroute.junit.wrapper;version='[4.12.0,4.12.1)',\
+	org.openhab.core;version='[2.5.0,2.5.1)',\
+	org.openhab.core.config.core;version='[2.5.0,2.5.1)',\
+	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
+	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
+	org.apache.commons.collections;version='[3.2.1,3.2.2)',\
+	org.objenesis;version='[2.6.0,2.6.1)',\
+	org.openhab.core.config.discovery;version='[2.5.0,2.5.1)',\
+	org.openhab.core.io.console;version='[2.5.0,2.5.1)',\
+	org.openhab.core.thing;version='[2.5.0,2.5.1)',\
+	slf4j.api;version='[1.7.25,1.7.26)',\
+	org.apache.commons.exec;version='[1.1.0,1.1.1)',\
+	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
+	org.eclipse.jetty.client;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.http;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.io;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.security;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.server;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.servlet;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.util;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.websocket.api;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.websocket.client;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.websocket.common;version='[9.4.11,9.4.12)',\
+	org.eclipse.jetty.xml;version='[9.4.11,9.4.12)',\
+	org.jupnp;version='[2.5.2,2.5.3)',\
+	org.openhab.binding.avmfritz;version='[2.5.0,2.5.1)',\
```

Comment:
That's supposed to be "tradfri" I guess

Reply:
Yes, of course. But currently this bndrun file is crappy anyways as long as I am not able to run the resolver.

Reply:
I'll remind you of this file as soon as we are tycho free and the resolver is working again ;)